Makefile.am

```
@@ -246,6 +262,16 @@ total.coverage/.dirstamp: total_coverage.info
 
 cov: test_bitcoin.coverage/.dirstamp cov-zen total.coverage/.dirstamp
 
+COMMIT = `git log -1 --format="%H"`
+
+cov_ci: test_bitcoin.coverage/.dirstamp cov-zen total.coverage/.dirstamp
+	bash <(curl -Ls https://coverage.codacy.com/get.sh) report --partial -l CPP --commit-uuid ${COMMIT} \
+    --api-token ${CODACY_API_TOKEN_COVERAGE} --organization-provider "gh" \
+    --username "HorizenOfficial" --project-name "zen" -r total_coverage.info 
+	bash <(curl -Ls https://coverage.codacy.com/get.sh) report --partial -l C   --commit-uuid ${COMMIT} \
+    --api-token ${CODACY_API_TOKEN_COVERAGE} --organization-provider "gh" \
+    --username "HorizenOfficial" --project-name "zen" -r total_coverage.info
+
```

I'd suggest the following to get rid of some duplication

```suggestion
cov_ci_%:
      bash <(curl -Ls https://coverage.codacy.com/get.sh) report --partial -l $* --commit-uuid ${COMMIT} \
    --api-token ${CODACY_API_TOKEN_COVERAGE} --organization-provider "gh" \
    --username "HorizenOfficial" --project-name "zen" -r total_coverage.info 

cov_ci: test_bitcoin.coverage/.dirstamp cov-zen total.coverage/.dirstamp
	cov_ci_CPP
	cov_ci_C
```

---

agree, modified!

---

Actually, this has been reverted to the original version